### DISCUSSION TITLE
How can I use span log data ?

### DISCUSSION THREAD

**Discussion by zhylinskiy-stryber** [jaegertracing/jaeger#2806] [Q&A] [answered]
Hello. 
Is there a way to utilise span log data ? With Prometheus or any other tool ?
For example compare certain key value with some average.

Thank you.

**Comment by jpkrohling** [comment, marked as answer]
No, you can use the search to look for spans with logs containing specific strings, but you can't aggregate spans based on it.

**Reply by zhylinskiy-stryber** [reply]
Thank you
